packages/wallets/src/features/cfd/modals/DxtradeEnterPasswordModal/DxtradeEnterPasswordModal.tsx

```diff
@@ -174,13 +180,16 @@ const DxtradeEnterPasswordModal = () => {
                     marketType='all'
                     onPasswordChange={e => setPassword(e.target.value)}
                     onPrimaryClick={onSubmit}
-                    onSecondaryClick={() =>
+                    onSecondaryClick={() => {
+                        sendEmail({
+                            platform: dxtradePlatform,
```

Comment:
what happens if sending email errors out or takes prolonged time?

Reply:
Thank you for pointing this out @wojciech-deriv  I'll look into it.